db/migrate/20151021093644_set_correct_sti_type_on_cloud_network.rb

```diff
@@ -15,7 +15,7 @@ class CloudNetwork < ActiveRecord::Base
     self.inheritance_column = :_type_disabled
 
     belongs_to :ext_management_system,
-               :foreign_key => :ems_id, :class_name => SetCorrectStiTypeOnCloudNetwork::BaseManager
+               :foreign_key => :ems_id, :class_name => "SetCorrectStiTypeOnCloudNetwork::BaseManager"
```

Comment:
Autoload is a constant problem... it now raises in rails 5.2 for `:class_name`  🎉